Documentation/bpf.rst

```diff
@@ -3008,22 +3263,227 @@ are mainly targeted for kernel developers involved in the BPF subsystem.
 Projects using BPF
 ------------------
 
-The following list includes some open source projects making use of BPF:
-
-- BCC - tools for BPF-based Linux IO analysis, networking, monitoring, and more
-  (https://github.com/iovisor/bcc)
-- Cilium
-  (https://github.com/cilium/cilium)
-- iproute2 (ip and tc tools)
-  (https://wiki.linuxfoundation.org/networking/iproute2)
-- perf tool
-  (https://perf.wiki.kernel.org/index.php/Main_Page)
-- ply - a dynamic tracer for Linux
-  (https://wkz.github.io/ply)
-- Go bindings for creating BPF programs
-  (https://github.com/iovisor/gobpf)
-- Suricata IDS
-  (https://suricata-ids.org)
+The following list includes a selection of open source projects making
+use of BPF respectively provide tooling for BPF. In this context the eBPF
+instruction set is specifically meant instead of projects utilizing the
+legacy cBPF:
+
+**Tracing**
+
+* **BCC**
+
+  BCC stands for BPF Compiler Collection and its key feature is to provide
+  a set of easy to use and efficient kernel tracing utilities all based
+  upon BPF programs hooking into kernel infrastructure based upon kprobes,
+  kretprobes, tracepoints, uprobes, uretprobes as well as USDT probes. The
+  collection provides close to hundred tools targeting different layers
+  across the stack from applications, system libraries, to the various
+  different kernel subsystems in order to analyze a system's performance
+  characteristics or problems. Additionally, BCC provides an API in order
+  to be used as a library for other projects.
+
+  https://github.com/iovisor/bcc
+
+..
+
+* **bpftrace**
+
+  bpftrace is a DTrace-style dynamic tracing tool for Linux and uses LLVM
+  as a back end to compile scripts to BPF-bytecode and makes use of BCC
+  for interacting with the kernel's BPF tracing infrastructure. It provides
+  a higher-level language for implementing tracing scripts compared to
+  native BCC.
+
+  https://github.com/ajor/bpftrace
+
+..
+
+* **perf**
+
+  Linux' perf tool which is developed by the Linux kernel community as
```

Comment:
`Linux's` ? I guess? I think this was how you referred to it elsewhere in the doc.